src/types/react-table.d.ts

```diff
@@ -60,6 +60,15 @@ interface CustomColumnProps {
   configurable?: boolean,
   primary?: boolean,
   label?: string,
+  bkColumnWidth?: {
+    flex: number,
+    width: string,
+  },
+}
+
+interface CustomTableConfig {
+  identifierColumnConfig?: { columnId: string, sticky?: boolean },
+  actionColumnConfig?: { columnId: string, sticky?: boolean },
```

Comment:
I don't think having a `columnId` config here would be useful. The "identifier" column must always be the first column, otherwise the sticky behavior doesn't make sense. And similarly the "actions" column must be the last column. So I think it would be sufficient for the user to specify (1) whether the first column should be sticky, and (2) whether the last column should be sticky.